course/frontend/hangman/index.html

```diff
@@ -7,7 +7,7 @@
     <title>Document</title>
```

Comment:
WARN! this content replaces a newer one! may branch be reset to upstream/develop first, and then only add your changes (which in any case should replace previously existing ones). HINT ask HELP to didac! he already did it in his branch!